### PR TITLE
Venue

### PR DESCRIPTION
**Pull request by Melisa-Zorraindo**
The user can now visit the specific venue to get further information.

Larger screens:
image

Smaller screens:
image

### CODE REVIEW

File: src/pages/venue/index.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        const response = await fetch(
+          `${baseUrl}/holidaze/venues/${id}?_owner=true`
+        );
+        const result = await response.json();
+
+        setVenue(result);
+      } catch (error) {
+        console.error(error);
+        setIsError(true);
```
Comment (chrislewispac) [inline src/pages/venue/index.tsx:38]: should we be exposing the actual error to the user rather than logging it?

File: src/pages/venue/index.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      </div>
+    );
+  }
+
+  if (isError) {
+    return <h1>An error occurred</h1>;
+  }
+
+  return (
+    venue && (
```
Comment (chrislewispac) [inline src/pages/venue/index.tsx:60]: wouldnt not having a venue be an erorr state in this case? maybe a 404 not found? rather than just an empty page.

**Review by chrislewispac — APPROVED** [review]

File: src/pages/venue/index.tsx (hunk 1 above)
Comment (Melisa-Zorraindo) [reply src/pages/venue/index.tsx:38]: defo, but I'm planning to make another PR for success/error handling and there'll be one api hook to handle calls and responses

File: src/pages/venue/index.tsx (hunk 2 above)
Comment (Melisa-Zorraindo) [reply src/pages/venue/index.tsx:60]: yes, you're right. I'll modify the code

**Pull request merged**